OracleHTTPServer/dockerfiles/14.1.2.0.0/container-scripts/stop-ohs.py

```diff
@@ -24,7 +24,7 @@
         sys.exit("Error: Please set the environment variable ORACLE_HOME")
 except (KeyError), why:
     sys.exit("Error: Missing Environment Variables " + str(why))
-nmConnect(username,password,domainName=domain_name)
+nmConnect(username,password,domainName=domain_name,domainDir=domain_path,host='localhost',port=5556, nmType='ssl')
```

Comment:
Have you tested this syntax of nmConnect?

Reply:
yes it has been tested and OHS successfully runs